### PR TITLE
Support for SLIM-labeling in FeatureFinderIdentification

### PR DESCRIPTION
**Pull request by hendrikweisser**
Adds a parameter ("model:each_trace") to FFId which enables fitting of elution models (Gaussian or EGH) to each mass trace in a feature. This is useful for data analysis in SLIM-labeling experiments (https://pubmed.ncbi.nlm.nih.gov/28821603), where intensities of different isotopologue peaks are compared.

### CODE REVIEW

File: src/openms/source/TRANSFORMATIONS/FEATUREFINDER/ElutionModelFitter.cpp [hunk 1]
```diff
@@ -113,16 +116,97 @@ double ElutionModelFitter::calculateFitQuality_(const TraceFitter* fitter,
...
+  // record model parameters:
+  double center = fitter->getCenter(), height = fitter->getHeight();
+  feature.setMetaValue("model_height", height);
+  feature.setMetaValue("model_FWHM", fitter->getFWHM());
+  feature.setMetaValue("model_center", center);
+  feature.setMetaValue("model_lower", fitter->getLowerRTBound());
+  feature.setMetaValue("model_upper", fitter->getUpperRTBound());
+  if (asymmetric)
+  {
+    EGHTraceFitter* egh = static_cast<EGHTraceFitter*>(fitter);
```
Comment (timosachsenberg) [inline src/openms/source/TRANSFORMATIONS/FEATUREFINDER/ElutionModelFitter.cpp]: these casts smell bad but not sure if its worth to refactor the TraceFitter class

**Comment by timosachsenberg** [comment]
looks good!

File: src/openms/source/TRANSFORMATIONS/FEATUREFINDER/ElutionModelFitter.cpp (hunk 1 above)
Comment (hendrikweisser) [reply src/openms/source/TRANSFORMATIONS/FEATUREFINDER/ElutionModelFitter.cpp]: That's how it's been before, so maybe an issue for another time.

**Comment by hendrikweisser** [comment]
Test failures on Windows seem unrelated (InspectAdapter failed to write index file), so good to merge?

**Pull request merged**